### DISCUSSION TITLE
Functionality similar to ConvLSTM or ConvGRU

### DISCUSSION THREAD

**Discussion by ChidanandKumarKS** [lava-nc/lava#196] [Q&A] [answered]
Do we have any components of spiky networks for ConvLSTM and ConvGRU? Kindly reply

**Comment by mgkwill** [comment, marked as answer]
@ChidanandKumarKS Thanks for your question. Let me check on it. Generally it is better to ask such questions in the https://github.com/lava-nc/lava/discussions/categories/q-a

I've converted it to a Q&A.

**Reply by ChidanandKumarKS** [reply]
Any modules updated in Lava on GonvGRU or ConvLSTM

**Reply by bamsumit** [reply]
There is no spiking equivalent of LSTM or GRU. The recurrent alternatives for SNNs are [LSNN](https://proceedings.neurips.cc/paper/2018/file/c203d8a151612acf12457e4d67635a95-Paper.pdf) and [LMU](https://papers.nips.cc/paper/2019/file/952285b9b7e7a1be5aa7849f32ffff05-Paper.pdf).

**Reply by ChidanandKumarKS** [reply]
@bamsumit , thanks for your reply. Will go through the referred links
